src/shared/src/arinc429.ts

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable max-classes-per-file */
+type SignStatusMatrixRange = (typeof Arinc429Word.SignStatusMatrix)[keyof typeof Arinc429Word.SignStatusMatrix];
+
+export class Arinc429WordSsmParseError extends Error {
+    constructor(public ssm: number) {
+        super();
+    }
+}
+
+export class Arinc429Word {
+    static SignStatusMatrix = Object.freeze({
+        FailureWarning: 0b00,
+        NoComputedData: 0b01,
+        FunctionalTest: 0b10,
+        NormalOperation: 0b11,
+    } as const);
+
+    static f64View = new Float64Array(1);
+
+    static u32View = new Uint32Array(Arinc429Word.f64View.buffer);
+
+    static f32View = new Float32Array(Arinc429Word.f64View.buffer);
+
+    ssm: SignStatusMatrixRange;
+
+    value: number;
+
+    constructor(word: number) {
```

Comment:
**suggestion**: As this is now also usable outside of a React context, consider adding the following functions (as found in `arinc429.js`) to make it easier to use:
```typescript
    static empty() {
        return new Arinc429Word(0);
    }

    static fromSimVarValue(name) {
        return new Arinc429Word(SimVar.GetSimVarValue(name, "number"));
    }
```